tests/devices/test_default_qubit_torch.py

```diff
@@ -214,9 +235,9 @@ def test_qubit_state_vector(self, init_state, tol):
         assert isinstance(res, torch.Tensor)
         assert torch.allclose(res, expected, atol=tol, rtol=0)
 
-    def test_full_subsystem_statevector(self, mocker):
+    def test_full_subsystem_statevector(self, device, torch_device, mocker):
         """Test applying a state vector to the full subsystem"""
-        dev = DefaultQubitTorch(wires=["a", "b", "c"])
+        dev = device(wires=["a", "b", "c"])
```

Comment:
Why are some tests applied on all `torch_device`'s, but others like this one aren't?

Reply:
Good catch! All should be applied on a device that took `torch_device`.